app/components/nav-bar.js

```diff
@@ -8,5 +8,13 @@ export default Component.extend({
       this.get('authManager').logout();
       this.get('routing').transitionTo('index');
     }
+  },
+
+  didInsertElement() {
```

Comment:
You need to clean up once done. 
Destroy the popup before the component is removed.